crates/core/Cargo.toml

```diff
@@ -6,21 +6,22 @@ description.workspace = true
 homepage.workspace = true
 
 [dependencies]
-async-trait = "0.1.58"
+async-trait.workspace = true
```

Comment:
is the rule for workspaces something like:

- any dependency that is shared across crates in a project with the same parameterization -> workspace?

Reply:
yes